src/Logger/Formatter/JsonFormatter.php

```diff
@@ -9,7 +9,11 @@
 class JsonFormatter extends BaseJsonFormatter
 {
     /**
-     * {@inheritdoc}
+     * Formats a log record.
+     *
+     * @param mixed[] $record A record to format
+     *
+     * @return mixed The formatted record
```

Comment:
This look like it it'll be a string (or possibly null?)

Reply:
You're right, updated